src/ESPEasyRules.ino

```diff
@@ -202,6 +243,25 @@ String rulesProcessingFile(const String& fileName, String& event) {
   return "";
 }
 
+void replace_EventValueN_Argv(String& line, const String& argString, unsigned int argc)
```

Comment:
maybe count from 0 to 4 and for 0 don't do "eventvalue += argc;" but only assign "argc =1;"

Reply:
Hmm that's a good one.
Will do :)